### DISCUSSION TITLE
Where does the processed video gets saved?

### DISCUSSION THREAD

**Discussion by Ryugon07** [roboflow/supervision#1075] [Q&A] [answered]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar feature requests.

### Question

Apologies for asking such a newbie question. Where does the processed file gets saved after running the inference_file_example.py script? I am using vs code to run this locally on my cpu. I can see the video as it was processing but as soon as its finished I couldn't find it anywhere.
I am using time_in_zone

Thanks

**Comment by SkalskiP** [comment]
Hi @Ryugon07 👋🏻 Let me convert this question into a discussion and move it to the Q&A section.

**Comment by SkalskiP** [comment, marked as answer]
Hi @Ryugon07 it does not get saved. But you can easily save it. Just add [`sv.VideoSink`](https://supervision.roboflow.com/develop/utils/video/#supervision.utils.video.VideoSink) to your code.

**Reply by Ryugon07** [reply]
Thanks @SkalskiP

**Reply by SkalskiP** [reply]
@ Ryugon07, we could add this logic to our examples and make them save the result video.
